server/src/Slushy.ts

```diff
@@ -10,7 +10,7 @@ export class Slushy {
     public constructor(
         public readonly props: Readonly<SlushyProps>,
         public readonly app: SlushyApplication = SlushyApplicationFactory.create(),
-        public readonly router: SlushyRouter = new SlushyRouter(props, app)
```

Comment:
Do we modify it now? Probably an oversight :)

Reply:
Found out another way to get it done. If you look inside `RegisterAuthenticationMiddleware` you'll that I create a new `express.Router`. That way I just register a `router` in a `router. Xzibit would be proud of me.

Reply:
Haha nice, hacking express 🥇